_posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey

```diff
@@ -0,0 +1,28 @@
+---
+author:
```

Comment:
Looks like this is missing the `layout: post`.

Reply:
@MishaCienfuegos went back and fixed this, thank you!

Reply:
Thank you for the catch @MishaCienfuegos, went back and added this.